README.md

```diff
@@ -0,0 +1 @@
+thats a "readme" file, but I didn't made enough for making "readme"
```

Comment:
Fix README. It doesn't contain information about your service, instructions on how to run it locally, etc. But it should.